ts/Extensions/RangeSelector.ts

```diff
@@ -428,6 +428,7 @@ extend(defaultOptions, {
          * @sample {highstock} stock/rangeselector/enabled/
          *         Disable the range selector
          *
+         * @type {boolean}
```

Comment:
There's now a mismatch between the doclet `@type` and the actual TypeScript type. I think it should be `{boolean|undefined}`. 

https://github.com/highcharts/highcharts/blob/ea08706707c0532d71e0398b93ec9bc331c8a28e/ts/Extensions/RangeSelector.ts#L116